bin/check_google_doc_export.rb

```diff
@@ -16,12 +16,21 @@
 require_relative '../lib/article_json'
 
 doc_id = '1E4lncZE2jDkbE34eDyYQmXKA9O26BHUiwguz4S9qyE8'
-url = "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
+url =
+  "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
 expected_document = JSON.parse(File.read('spec/fixtures/reference_document_parsed.json'))
 
 exported_doc = Net::HTTP.get(URI.parse(url))
 document = JSON.parse(ArticleJSON::Article.from_google_doc_html(exported_doc).to_json)
 
+# `source_url` (for hosted images) is dynamic, so we need to remove it from the comparison`
+def nullify_source_url(hash)
+  hash['content'].each { |key, _value| key['source_url'] = nil if key['source_url'] }
+end
+
+expected_document = nullify_source_url(expected_document)
```

Comment:
The `nullify_source_url(expected_document)` call returns `expected_document['content']` not `expected_document` itself as far as I understand, is this what we need here? Because the variables names here are a bit confusing to me.